src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exseption.InvalidAgeException;
+import core.basesyntax.exseption.InvalidLoginException;
+import core.basesyntax.exseption.InvalidPasswordException;
+import core.basesyntax.exseption.LoginIsTakenException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static String VALID_LOGIN_1;
+    private static String VALID_LOGIN_2;
+    private static String VALID_PASSWORD;
+    private static String INVALID_PASSWORD;
+    private static Integer AGE_OVER_18;
+    private static Integer AGE_UNDER_18;
+    private static Integer AGE_18;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        VALID_LOGIN_1 = "first_person";
+        VALID_LOGIN_2 = "second_person";
+        VALID_PASSWORD = "12345678";
+        INVALID_PASSWORD = "12345";
+        AGE_OVER_18 = 22;
+        AGE_UNDER_18 = 17;
+        AGE_18 = 18;
+    }
+
+    @Test
+    void userWithValidParameters() {
```

Comment:
make all your test methods' names consistent, use this pattern 
<img width="825" alt="image" src="https://user-images.githubusercontent.com/96740815/197988777-e0f22940-6216-4226-9e1a-f1f42e282c62.png">